sources/editor/Stride.GameStudio.Avalonia/App.axaml.cs

```diff
@@ -31,7 +32,7 @@ public override void OnFrameworkInitializationCompleted()
 
         if (ApplicationLifetime is IClassicDesktopStyleApplicationLifetime desktop)
         {
-            desktop.MainWindow = new MainWindow();
+            desktop.MainWindow = new ProjectSelectionWindow();
```

Comment:
I'm not quite decided about that one. I know the WPF version does start with this window, but I don't like it. I'd prefer we change the logic and start with the main window. We could force open the new/select window after the main one is created if needed.

I made sure that a new session can be opened after an already loaded one. So the trick to basically restart everything for a new project isn't needed anymore. At least for now.